<launch>
  <!-- service -->
  <node pkg="iri_tos_supervoxels" type="iri_tos_supervoxels"
   name="iri_tos_supervoxels_alg" output="screen">
      <param name="seed_resolution" value="0.015"/>
      <param name="zmin" value="0.02"/>
      <param name="th_points" value="50"/>
      <param name="service_name" value="object_segmentation"/>
  </node>

</launch>